app.js

```diff
@@ -53,13 +57,18 @@ app.get("/api/users/:user_id/reviews", getUserReviews);
 app.get("/api/users/:user_id/orders", getUserOrders);
 app.patch("/api/users/:user_id/reviews/:review_id", patchUserReview);
 app.patch("/api/users/:user_id/orders/:order_id", patchOrder);
-app.post("/api/users", postUser);
 app.post("/api/users/:user_id/reviews", postReview);
 app.post("/api/users/:user_id/orders", postOrder);
 app.delete("/api/users/:user_id", deleteUser);
 app.delete("/api/:user_id/reviews/:review_id", removeReview);
 app.delete("/api/users/:user_id/orders/:order_id", removeOrder);
 
+app.post("/api/users/register", registerUser);
```

Comment:
Why are we changing this endpoint name?

Reply:
was basically to differntiate between api/users/reg and api/tech/reg if we going down that route. It also reflects better on what the end point is doing.